tests/rustdoc-gui/source-anchor-scroll.goml

```diff
@@ -8,13 +8,13 @@ set-window-size: (600, 800)
 assert-property: ("html", {"scrollTop": "0"})
 
 click: '//a[text() = "barbar"]'
-assert-property: ("html", {"scrollTop": "125"})
+assert-property: ("html", {"scrollTop": "149"})
```

Comment:
Why did these values change? Unless you changed the default size of the page, they should not have been updated.

Reply:
I see, thanks! But why generating the links to def all the time then? Was there an issue to generate it only for one crate? It's a nightly-only option, so it should not be included all the time because it prevents us to test the almost 100% use-case, which is problematic imo.